Sources/HTTP/Models/Client/HTTP+Client.swift

```diff
@@ -24,54 +24,75 @@ public final class Client<
     public let host: String
     public let port: Int
     public let securityLayer: SecurityLayer
+    public let middleware: [Middleware]
 
     public let stream: Stream
 
-    public init(scheme: String, host: String, port: Int, securityLayer: SecurityLayer) throws {
+    private let responder: Responder
+
+    public init(
+        scheme: String,
+        host: String,
+        port: Int = 80,
+        securityLayer: SecurityLayer = .none,
+        middleware: [Middleware] = []
```

Comment:
Same here, I think it should be `= WhateverType.defaultMiddleware`.

If the protocol typing gets weird for default, I think just a global `defaultMiddleware` would be better to provide by default. Especially for how often the 1-off client stuff is used.